File: Java/String/Right.java
class Program {
	public static void main (String[] args) {
		System.out.println(Right("OlaMundo", 3));
	}

	public static String Right(String text, int length) {
		if (text.length() <= length) return null;
		return text.substring(text.length() - length);
	}
}

//https://pt.stackoverflow.com/q/349464/101

File: CSharp/String/Right.cs
using System;
using static System.Console;

public class Program {
	public static void Main() {
		WriteLine("123456789".Right(4));
	}
}

namespace System {
    public static class StringExtensions {
        public static string Right(this string texto, int length) => texto.Substring(texto.Length - length);
    }
}

//https://pt.stackoverflow.com/q/349829/101
